pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -160,6 +161,16 @@ func (c *Cluster) Start() error {
 		}
 	}
 
+	mgrsToSkipReconcile, err := c.getMGRsToSkipReconcile()
+	if err != nil {
+		return errors.Wrap(err, "failed to check for mgrs to skip reconcile")
+	}
+
+	if mgrsToSkipReconcile.Len() > 0 {
```

Comment:
Instead of skipping if any are labeled, it would be ideal to skip only the ones labeled. The OSDs follow this pattern of only skipping the labeled ones. The mons just don't follow that pattern, but I think it's best for all other daemons.

Reply:
I think it would be ok to follow the same pattern like mon for mgr, mds and rgw.  User might not want to upgrade the standby mds while debugging it active mds